tests/testthat/test-plot_ggonion.R

```diff
@@ -6,3 +6,11 @@ testthat::test_that("plot ggonion", {
   clr <- c("red","yellow", "green")
   ggonion(x, ratio = 2, bias = 0, color = clr)
 })
+testthat::test_that("plot ggonion with themes", {
```

Comment:
Actually, this is not a test , test should check some expectations for real code behavior. 
Please, check https://journal.r-project.org/archive/2011/RJ-2011-002/RJ-2011-002.pdf